File: put.pl
use strict;
use LWP::UserAgent;
use HTTP::Request::Common;


# URLを変更してください
my $url = 'https://9bqcyrr7bb.execute-api.ap-northeast-1.amazonaws.com/dev/Q14';
# POSTするデータを書き換えてください
my %postdata = ( 'id' => 'id', 'pass' => 'pass' );

# 一つなら
#my %postdata = ('id'=>'id')
#
my $request = POST( $url, \%postdata );

# 送信
my $ua = LWP::UserAgent -> new;
my $res = $ua -> request( $request ) -> as_string;

File: put.py
import requests

#URLを変更してください
url = "https://9bqcyrr7bb.execute-api.ap-northeast-1.amazonaws.com/dev/Q15"
s = requests.session()
# POSTするデータを変更してください
params = {"id": "aaa","passwd": "bbbbb"}
r =  s.post(url, data=params)
print (r.text.encode("utf-8"))
